### DISCUSSION TITLE
[Feature Request] 请在此填写标题

### DISCUSSION THREAD

**Discussion by huilingaa** [youzan/vant#11369] [Q&A]
### 这个功能解决了什么问题？

在pc端也页面整了个手机端的div，模拟移动端，用了vant的组件，怎么适配

### 你期望的 API 是什么样子的？

可以以某个div为参考标准，去做

**Comment by chenjiahan** [comment]
没理解你的意思，看起来不是一个 Feature Request，先移动到讨论区了。
